tests/TestUtility/SharedOmniSharpHostFixture.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace TestUtility
+{
+    public class SharedOmniSharpHostFixture : IDisposable
+    {
+        public SharedOmniSharpHostFixture()
+        {
+        }
+
+        public void Dispose()
```

Comment:
Doesn't look like we're calling this anywhere-does this happen automatically though some xunit magic?

Reply:
correct, it's an [xunit feature](https://xunit.github.io/docs/shared-context.html)
`Dispose()` in a test class will run at the end of each test, while `Dispose()` in a fixture class will run at the end of the collection.